code/game/machinery/deployable.dm

```diff
@@ -63,6 +60,14 @@
 	bar_material = WOOD
 	var/drop_amount = 3
 
+/obj/structure/barricade/wooden/Initialize(mapload)
+	. = ..()
+
+	if(!(flags_1 & NODECONSTRUCT_1))
```

Comment:
Why are we checking for this flag if crowbar_act doesn't check for any of this and works regardless of it?

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/69676/commits/2e650bcfcb7f285fe3d9bc4e4019c8d61d7157ba

I saw there was a check for it in:

```
/obj/structure/barricade/deconstruct(disassembled = TRUE)
	if(!(flags_1 & NODECONSTRUCT_1))
		make_debris()
	qdel(src)
```

So I assumed there were some barricades that shouldn't be deconstructable.